From 6aeacfcb1bd474545cd4ad5f1cd94022bf5b8b66 Mon Sep 17 00:00:00 2001
From: Tim Gover <tim.gover@raspberrypi.com>
Date: Wed, 3 Apr 2024 17:44:44 +0100
Subject: [PATCH 1002/1085] dts: bcm2712: cm5: Disable HS400

Disable HS400 support on CM5 until this is proved to be stable.

Signed-off-by: Tim Gover <tim.gover@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi | 1 -
 1 file changed, 1 deletion(-)

--- a/arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi
+++ b/arch/arm/boot/dts/broadcom/bcm2712-rpi-cm5.dtsi
@@ -353,7 +353,6 @@ dpi_16bit_gpio2:        &rp1_dpi_16bit_g
 	sd-uhs-ddr50;
 	sd-uhs-sdr104;
 	mmc-hs200-1_8v;
-	mmc-hs400-1_8v;
 	broken-cd;
 	supports-cqe;
 	status = "okay";
